core/src/main/java/jenkins/util/DirectedGraph.java

```diff
@@ -76,6 +77,7 @@ public int size() {
          */
         int lowlink;
 
+        @SuppressFBWarnings(value = "URF_UNREAD_FIELD", justification = "no big deal")
```

Comment:
Technically unused, but useful when reading the code and following along in Tarjan's algorithm.